api/identifiers.proto

```diff
@@ -95,3 +95,11 @@ message EntityIdentifiers {
   }
 }
 
+// Identifies an end device model with version information.
+message EndDeviceVersionIdentifiers {
+  string brand_id = 1 [(gogoproto.customname) = "BrandID", (validate.rules).string = {pattern: "^[a-z0-9](?:[-]?[a-z0-9]){2,}$", max_len: 36, ignore_empty: true}];
```

Comment:
Nit: Remove `customname`. Same for `model_id`.

Reply:
This pretty much means that all `.ModelID` references will need to be updated to `ModelId`, which is a rather large diff in the context of this PR. I think it is better to make this change in a separate PR (since it touches multiple components).

Reply:
That's fine with me, but please keep in mind that we're in the process of phasing out `gogoproto`, so it's actually ok to do such modifications, given that they are just renames.

Reply:
I'm fine with keeping that out of scope for this PR.